extensions/wikia/Search/classes/Services/QuestDetails/QuestDetailsSolrHelper.class.php

```diff
@@ -166,7 +175,7 @@ protected function getAbstract( $item ) {
 	 */
 	protected function getMetadata( $metadata ) {
 
-		$output = [ "map_location" => [ ] ];
+		$output = [ "map_location" => [] ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "map_location" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)